smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/http-binding-trait-ignored.errors

```diff
@@ -0,0 +1,14 @@
+[ERROR] smithy.example#IgnoredBindingOperationInput$code: Trait `httpResponseCode` cannot be applied to `smithy.example#IgnoredBindingOperationInput$code`. This trait may only be applied to shapes that match the following selector: structure :not([trait|input]) > member :test(> integer) | TraitTarget
+[NOTE] smithy.example#IgnoredBindingOperationOutput: Structure member `stringMap` is marked with the `httpQueryParams` trait, and `httpQuery` traits are applied to the following members: `string2`. The service will not be able to disambiguate between query string parameters intended for the `stringMap` member and those explicitly bound to the `httpQuery` members. | HttpQueryParamsTrait
+[NOTE] smithy.example#NestedBindings: Structure member `stringMap` is marked with the `httpQueryParams` trait, and `httpQuery` traits are applied to the following members: `string2`. The service will not be able to disambiguate between query string parameters intended for the `stringMap` member and those explicitly bound to the `httpQuery` members. | HttpQueryParamsTrait
+[WARNING] smithy.example#IgnoredBindingOperationInput$code: This member has the `smithy.api#httpResponseCode` trait applied but its container has the following non-output relationships: [{INPUT=smithy.example#IgnoredBindingOperationInput}] | HttpBindingTraitIgnored.Output
+[WARNING] smithy.example#IgnoredBindingOperationOutput$string: This member has the `smithy.api#httpLabel` trait applied but its container has the following non-input relationships: [{OUTPUT=smithy.example#IgnoredBindingOperationOutput}] | HttpBindingTraitIgnored.Input
+[WARNING] smithy.example#IgnoredBindingOperationOutput$string2: This member has the `smithy.api#httpQuery` trait applied but its container has the following non-input relationships: [{OUTPUT=smithy.example#IgnoredBindingOperationOutput}] | HttpBindingTraitIgnored.Input
+[WARNING] smithy.example#IgnoredBindingOperationOutput$stringMap: This member has the `smithy.api#httpQueryParams` trait applied but its container has the following non-input relationships: [{OUTPUT=smithy.example#IgnoredBindingOperationOutput}] | HttpBindingTraitIgnored.Input
+[WARNING] smithy.example#NestedBindings$string: This member has the `smithy.api#httpLabel` trait applied but its container has the following non-input relationships: [{MEMBER_TARGET=smithy.example#NestedBindings}] | HttpBindingTraitIgnored.Input
```

Comment:
Can you simplify this to:
1. "The `httpLabel` trait applied to this member is ignored in all contexts." (when it isn't ever effective, but does get referenced)
2. "The `httpLabel` trait applied to this member is ignored in some contexts: ... list here in a nicer way